src/sentry/scim/endpoints/teams.py

```diff
@@ -292,11 +313,70 @@ def _rename_team_operation(self, request: Request, new_name, team):
                 data=team.get_audit_log_data(),
             )
 
+    @extend_schema(
+        operation_id="Update a Team's Attributes",
+        parameters=[GLOBAL_PARAMS.ORG_SLUG, SCIM_PARAMS.TEAM_ID],
+        request=SCIMTeamPatchRequestSerializer,
+        responses={
+            204: RESPONSE_SUCCESS,
+            401: RESPONSE_UNAUTHORIZED,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOTFOUND,
+        },
+    )
     def patch(self, request: Request, organization, team):
         """
         A SCIM Group PATCH request takes a series of operations to perform on a team.
         It does them sequentially and if any of them fail no operations should go through.
         The operations are add members, remove members, replace members, and rename team.
+        Update a team's attributes with a SCIM Group PATCH Request. Valid Operations are:
+        * Renaming a team:
```

Comment:
in our API docs we don't currently display multiple examples so unfortunately, these have to go into the description. 

Updating the doc site to have multiple examples will be an important feature to add.